account_invoice_negative_total/README.rst

```diff
@@ -0,0 +1,68 @@
+========================
+Beescoop  Account Module
```

Comment:
The name of the module is not correct. :)

Reply:
fixed